sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -4907,9 +4907,9 @@ def __setitem__(self, index, value):
 
 
 class _DeferredStringMethods(frame_base.DeferredBase):
-  @frame_base.with_docs_from(pd.core.strings.StringMethods)
-  @frame_base.args_to_kwargs(pd.core.strings.StringMethods)
-  @frame_base.populate_defaults(pd.core.strings.StringMethods)
+  @frame_base.with_docs_from(pd.Series.str)
```

Comment:
does this require a mininmal version of Pandas 1.x ?

Reply:
I suppose we will find out  from test results.

Reply:
This should be fine as of our minimums.